src/validator/validator.js

```diff
@@ -0,0 +1,31 @@
+const { check, validationResult } = require("express-validator");
+
+exports.registrationValidator = [
+  check("login")
+    .notEmpty()
+    .withMessage("Empty String")
+    .matches(/^[a-z]+([-_]?[a-z0-9]+){0,2}$/i),
```

Comment:
please read those task again, there is another validation
and write here too withMessage